http://bugs.gentoo.org/280248
Don't bind smtp-end-of-line. Patch from upstream CVS, by Yoichi NAKAYAMA.

--- wl/wl-draft.el	28 Jan 2006 09:03:41 -0000	1.258
+++ wl/wl-draft.el	9 Feb 2006 03:57:39 -0000	1.259
@@ -147,10 +147,6 @@
 	     (if (eq wl-smtp-connection-type 'ssl)
 		 #'open-ssl-stream
 	       smtp-open-connection-function))
-	    (smtp-end-of-line
-	     (if (eq wl-smtp-connection-type 'ssl)
-		 "\n"
-	       smtp-end-of-line))
 	    smtp-sasl-user-name smtp-sasl-properties sasl-read-passphrase)
        (setq smtp-sasl-user-name wl-smtp-posting-user
 	     smtp-sasl-properties (when wl-smtp-authenticate-realm
